content/scalability/docs/workloads.md

```diff
@@ -0,0 +1,89 @@
+# Workloads
+
+Workloads have an impact on how large your cluster can scale. Workloads that use the Kubernetes APIs heavily will limit the total amount of workloads you can have in a single cluster, but there are some defaults you can change to help reduce the load.
+
+Workloads in a Kubernetes cluster have access to features that integrate with the Kubernetes API (e.g. Secrets and ServiceAccounts), but these features are not always required and should be disabled if they’re not being used. Limiting workload access and dependence on the Kubernetes control plane will increase the number of workloads you can run in the cluster and improve the security of your clusters by removing unnecessary access to workloads and implementing least privilege practices. Please read the [security best practices](https://aws.github.io/aws-eks-best-practices/security/docs/) for more information.
+
+## Use IPv6 for pod networking
+
+You cannot transition a VPC from IPv4 to IPv6 so enabling IPv6 before provisioning a cluster is important. If you enable IPv6 in a VPC it does not mean you have to use it and if your pods and services use IPv6 you can still route traffic to and from IPv4 addresses. Please see the [EKS networking best practices](https://aws.github.io/aws-eks-best-practices/networking/index/) for more information.
+
+Using [IPv6 in your cluster](https://docs.aws.amazon.com/eks/latest/userguide/cni-ipv6.html) avoids some of the most common cluster and workload scaling limits. IPv6 avoids IP address exhaustion where pods and nodes cannot be created because no IP address is available. It also has per node performance improvements because pods receive IP addresses faster by reducing the number of ENI attachments per node. You can achieve similar node performance by using [IPv4 prefix mode in the VPC CNI](https://aws.github.io/aws-eks-best-practices/networking/prefix-mode/), but you still need to make sure you have enough IP addresses available in the VPC.
+
+## Limit number of services per namespace
+
+The maximum number of services in a namespaces is 5,000 and 10,000  in a single cluster. To help organize workloads and services, increase performance, and to avoid cascading impact for namespace scoped resources we recommend limiting the number of services per namespace to 500.
```

Comment:
How can max services be both 5k and 10k?

Reply:
Can we re-word that sentence. I had to read it a couple times before it made sense, e.g. "The max number of services in a namespace is 5k and the max services per cluster is 10k". Also, who set these maximums? The project? We should cite the source.